app/presenters/update_password_presenter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class UpdatePasswordPresenter
+  attr_reader :user, :required_password_change
+
+  alias_method :required_password_change?, :required_password_change
+  def initialize(user:, required_password_change: false)
+    @user = user
+    @required_password_change = required_password_change
+  end
+
+  def forbidden_passwords
+    user.email_addresses.flat_map do |email_address|
+      ForbiddenPasswords.new(email_address.email).call
+    end.uniq
+  end
+
+  def aria_described_by_if_eligible
+    return {} if required_password_change?
```

Comment:
Since we render the description for all views, do we need this condition anymore, or should we just always apply the `aria-describedby` ?